wordpress-mobile/WordPress-iOS
# Support: Help Center search results 'Contact us' link

Fixes #12779

This toggles the 'Contact us' link on the Help Center search results view. If the user has set a 'Contact Email', it will be shown. If not, then it will not be shown.

To test:
- Start with a fresh install.
- Go to Me > Help & Support. 'Contact Email' should be 'Not Set'.
- Select 'WordPress Help Center'.
- Search for something that returns no results.
- Verify the 'Contact us' link does not appear.

<img width="400" alt="no_link" src="https://user-images.githubusercontent.com/1816888/68053524-c1a90000-fcb1-11e9-9bcc-8ced9147d85c.png">

---
- Close the Help Center.
- Set your 'Contact Email'.
- Select 'WordPress Help Center'.
- Search for something that returns no results.
- Verify the 'Contact us' link does appear.

<img width="400" alt="link" src="https://user-images.githubusercontent.com/1816888/68053552-d2597600-fcb1-11e9-9e1b-d3173915b213.png">

---
Update release notes:
